Applications/IRCClient/IRCWindow.h

```diff
@@ -75,7 +75,7 @@ class IRCWindow : public GUI::Widget {
     Type m_type;
     String m_name;
     RefPtr<Web::HtmlView> m_html_view;
-    RefPtr<GUI::TextEditor> m_text_editor;
+    RefPtr<GUI::TextBox> m_text_editor;
```

Comment:
<sergey> `m_text_editor` -> `m_text_box` </sergey>

...sorry, I can't help it :)

Reply:
lol invisible sergey tags

Reply:
I suppose I could have renamed it, but it had always been a TextBox anyways, just stored as a TextEditor for some reason, so I figured it best to leave it as it was. I can change it if that makes sense

Reply:
I dunno, I was just channeling @bugaevc, I don't know anything about naming things

Reply:
Well, the deed has been done!

Reply:
Hahah

I wouldn't have actually nitpicked about that though :smile:

Reply:
tch.
need more samples of things that wouldn't get you nitpicking.